m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh

```diff
@@ -118,7 +118,7 @@ unmount_and_clean()
 		rm -rf $MOTR_M0T1FS_TEST_DIR/d$ios_index/stobs/o/*
 	done
 
-        if [ ! -z $multiple_pools ] && [ $multiple_pools == 1 ]; then
+        if [ ! -z "$multiple_pools" ] && [ $multiple_pools == 1 ]; then
```

Comment:
This one is better to change to `[[ $multiple_pools == 1 ]]`. Without quotes, yes - because:
1. `[[ ... ]]` does not do word-splitting.
2. The `multiple_pools` variable is controlled by bash-scripts, not user inputs.

For more info refer to https://github.com/bahamas10/bash-style-guide#quoting.